yt/yt/core/actions/invoker.h

```diff
@@ -40,6 +40,15 @@ struct IInvoker
 
 DEFINE_REFCOUNTED_TYPE(IInvoker)
 
+struct IInvokerWithExpectedBytes
```

Comment:
It seems to me that it is not good to explicitly associate the invoker with the size in bytes. Perhaps this entity should be located next to the IO layer?

Reply:
> It seems to me that it is not good to explicitly associate the invoker with the size in bytes.

I agree. I wasn't able to find a better solution to pass expected bytes to the ThreadPool interface

> Perhaps this entity should be located next to the IO layer?
I agree